CHANGELOG.md

```diff
@@ -1,5 +1,26 @@
 # Changelog
 
+## [1.12.4] - 2025-06-26
+
+### Added
+
+- `k8s_configmap` BundlePublisher plugin (#6105, #6139)
+- UpstreamAuthority.SubscribeToLocalBundle RPC to stream updates in the local trust bundle (#6090)
+- Integration tests running on ARM64 platform (#6059)
```

Comment:
is this affecting users in some way? sounds like something internal

Reply:
As an end user of SPIRE running on ARM64, I would personally find it valuable to know that integration tests now run on ARM64 for each release. That said, I’m happy to remove this note if you think it’s too internal.